html-forms/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="index.html">
+      <div>
+        <label for="email-address">Email</label>
+      </div>
+      <div>
+        <input type="email" name="email" />
+      </div>
+
+      <div>
+        <label for="why-is-coding-cool">Why is coding cool?</label>
+      </div>
+      <div>
+        <select name="reasons">
+          <option value="select-an-option"> Select an option</option>
+          <option value="because-it's-interesting">Because it's interesting</option>
+          <option value="because-it's-fun">Because it's fun</option>
+          <option value="because-it's powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <br>
```

Comment:
Hi Justin ! please only use br elements when splitting up paragraphs of text onto new lines. Use divs to separate form elements onto new lines